src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,122 @@
+        if (r.next()) {
+            int id = r.getInt("id");
+            String password = r.getString("password");
+            return new Restaurant(id, restaurantText, isOpen, rating);
+        }
+        ;
+
+    public static void editRestaurant(Connection conn, String restaurantText) throws SQLException {
+        PreparedStatement stmt = conn.prepareStatement("UPDATE * FROM restaurants WHERE id = ?");
+        stmt.execute();
+    }
+
+    public static void main(String[] args) throws SQLException {
+
+        Server.createWebServer().start();
+        Connection conn = DriverManager.getConnection("jdbc:h2:~/main"); //connection object to particular database
+        Statement stmt = conn.createStatement();
+        stmt.execute("CREATE TABLE IF NOT EXISTS restaurants (id IDENTITY, name VARCHAR, is_open BOOLEAN, rating DOUBLE)");
+
+
+        Spark.init();
+        Spark.post(
+                "/insertRestaurant",
+                ((request, response) -> {
+
+                    String restaurantName = request.queryParams("restaurantName");
+                    Boolean restaurantOpen = Boolean.valueOf(request.queryParams("restaurantOpen"));
+                    double restaurantRating = Double.valueOf(request.queryParams("restaurantRating"));
+                    insertRestaurant(conn, restaurantName, restaurantOpen, restaurantRating);
+
+                    response.redirect("/");
+                    return "";
+                })
+        );
+
+        Spark.post(
+                "/deleteRestaurant",
+                ((request, response) ->
+                {
+                    public static void deleteRestaurant (Connection conn,int id) throws SQLException {
+                    PreparedStatement stmt = conn.prepareStatement("DELETE restaurant SET is_Open = False is_Open WHERE id = ?");
+                    stmt.setInt(1, id);
+                    stmt.execute();
+                }
+                };
+
+        Spark.get(
+                "/selectRestaurant",
+                ((request, response) -> {
+
+                    String restaurantName = request.queryParams("restaurantName");
+                    ArrayList<RestaurantList> (Connection conn) throws SQLException {
+                        ArrayList<RestaurantList> restaurants = new ArrayList<>();
+                        Statement stmt = conn.createStatement();
+                        ResultSet results = stmt.executeQuery("SELECT * FROM restaurants");
+                        while (results.next()) {
+                            int id = results.getInt("id");
+                            String text = results.getString("text");
+                            boolean isDone = results.getBoolean("is_Open");
+                            double isOpen = results.getDouble("rating");
+                            restaurants.add(new RestaurantList(id, text, isOpen, rating));
+                        }
+                        return restaurants;
+                    }
```

Comment:
Don't need this selectRestaurant route.  This should be a static method.  It should be called to display the restaurant list.